### ISSUE TITLE
Authentication Failed After login by google 

### ISSUE THREAD

**Issue by Yisaer** [vitinsinha/springboot-oauth2-facebook-google#1]
I replace my google client_id and client_secret in application.yml. And set in `http://localhost:3000/connect/google` as redirect_uri in google dev console.

After i run this application and click login with google , I choose my google account and finally got type=Unauthorized, status=401

Is this normal ?

**Comment by vitinsinha** [comment]
No, its not normal. I am not able to reproduce this issue. I followed these steps -
1. Cloned this project
2. Created a new google account
3. Created a new app in the new google account
4. Set the Authorized redirect URI as: http://localhost:3000/connect/google
5. Replaced google clientId and clientSecret in the application.yml file
6. Started the application and logged in with google. After login, it redirected me back to: http://localhost:3000/# and this text was displayed on the page: 
Logged in as: google_username
